backend/src/App.js

```diff
@@ -7,7 +7,7 @@ import { UsersRegistrationController } from './modules/UsersRegistration/control
 import { UserRegistrationService } from './modules/UsersRegistration/service/UserRegistration.service.js';
 import { MongoUsersRegistrationRepository } from './modules/UsersRegistration/repository/mongo/MongoUsersRegistration.repository.js';
 import { InMemoryUsersRegistrationRepository } from './modules/UsersRegistration/repository/inMemory/InMemoryUsersRegistration.repository.js';
-import { commentModule } from './modules/AddComment/CommentModule.js';
+import { CommentModule } from './modules/Comment/CommentModule.js';
```

Comment:
CommentModule.js eksportuje funkcję, to raczej chcemy: nazwę eksportowanej funkcji  małą literą.
```suggestion
import { commentModule } from './modules/Comment/CommentModule.js';
```